docs/python_docs/python/tutorials/packages/gluon/loss/loss.md

```diff
@@ -19,8 +19,8 @@
 
 Loss functions are used to train neural networks and to compute the difference between output and target variable. A critical component of training neural networks is the loss function. A loss function is a quantative measure of how bad the predictions of the network are when compared to ground truth labels. Given this score, a network can improve by iteratively updating its weights to minimise this loss. Some tasks use a combination of multiple loss functions, but often you'll just use one. MXNet Gluon provides a number of the most commonly used loss functions, and you'll choose certain loss functions depending on your network and task. Some common task and loss function pairs include:
 
-- regression: [L1Loss](/api/python/docs/api/gluon/_autogen/mxnet.gluon.loss.L1Loss.html), [L2Loss](/api/python/docs/api/gluon/_autogen/mxnet.gluon.loss.L2Loss.html) 
-- classification: [SigmoidBinaryCrossEntropyLoss](/api/python/docs/api/gluon/_autogen/mxnet.gluon.loss.SigmoidBinaryCrossEntropyLoss.html), [SoftmaxBinaryCrossEntropyLoss](/api/python/docs/api/gluon/_autogen/mxnet.gluon.loss.SoftmaxBinaryCrossEntropyLoss.html) 
+- regression: [L1Loss](/api/python/docs/api/gluon/_autogen/mxnet.gluon.loss.L1Loss.html), [L2Loss](/api/python/docs/api/gluon/loss/index.html#mxnet.gluon.loss.L2Loss) 
+- classification: [SigmoidBinaryCrossEntropyLoss](/api/python/docs/api/gluon/_autogen/mxnet.gluon.loss.SigmoidBinaryCrossEntropyLoss.html), [SoftmaxBinaryCrossEntropyLoss](/api/python/docs/api/gluon/_autogen/mxnet.gluon.loss.SoftmaxBinaryCrossEntropyLoss.html)
```

Comment:
```suggestion
- classification: [SigmoidBinaryCrossEntropyLoss](/api/python/docs/api/gluon/loss/index.html#mxnet.gluon.loss.SigmoidBinaryCrossEntropyLoss), [SoftmaxBinaryCrossEntropyLoss](/api/python/docs/api/gluon/loss/index.html#mxnet.gluon.loss.SoftmaxCrossEntropyLoss)
```

Reply:
Note: I'm not sure if the "Softmax Cross Entropy Loss" function is the same as the "Sigmoid Binary Cross Entropy Loss" function that it meant to be referenced in the second link.

Reply:
```suggestion
- classification: [SigmoidBinaryCrossEntropyLoss](/api/python/docs/api/gluon/loss/index.html#mxnet.gluon.loss.SigmoidBinaryCrossEntropyLoss), [SoftmaxCrossEntropyLoss](/api/python/docs/api/gluon/loss/index.html#mxnet.gluon.loss.SoftmaxCrossEntropyLoss)
```